apps/pie-docs/src/iconData.json

```diff
@@ -1697,8 +1697,8 @@
             "displayName": "Social",
             "icons": [
                 {
-                    "name": "facebook",
-                    "displayName": "Facebook"
+                    "name": "facebook-static",
+                    "displayName": "Facebook Static"
```

Comment:
Docs site question – do we want the display names to have static in them? (not sure if you strip that out, but may be better to strip it out when the icons get compiled)

Reply:
I didn't. Feels like a good indication for expected icon behaviour: that static variants won't change colour like non-static. For example, `facebook-circle` can change colour on hover/focus or during theme change etc while `facebook-static` can't.

Reply:
Sounds good - I don't mind either way. May be worth just flagging with @jordifornellsJET that it'll change the icon names like this once this gets merged

Reply:
I talked to Molly already. Will let Jordi know as well.